tests/e2e/cucumber/features/integrations/link.feature

```diff
@@ -31,6 +32,46 @@ Feature: link
     #When the public reloads the public link pages
     #Then the public should not see the following files on the files-drop page
     #  | textfile.txt |
-    When "Anonymous" logs out
-
+    And "Anonymous" logs out
+    When "Alice" logs in
```

Comment:
extra step can be removed with step 21 line

Reply:
as victor said, we do not have to log out alice in line 21 and as a result no re-login is needed